zh/customization.md

```diff
@@ -87,7 +87,7 @@ content/
 
 ## 评论
 
-Eureka支持使用Disqus，Utterances和Commento作为评论系统。如需配置评论，只需对`config/_default/params.yaml`中的`comment`进行相关配置即可。
+Eureka支持使用Disqus，Utterances，Valine和Commento作为评论系统。如需配置评论，只需对`config/_default/params.yaml`中的`comment`进行相关配置即可。
```

Comment:
Please add the detail explanation of valine. You can refer to `comment.utterances` or `diagram.mermaid` for example.